tailwind.config.ts

```diff
@@ -27,7 +27,11 @@ const config: Config = {
 			mono: "var(--mono-font)",
 			glb: "glacial",
 		},
-		extend: {},
+		extend: {
+      screens: {
+        ps : "375px",
```

Comment:
Is this screen-size necessary ? Try to avoid custom size and stick with tailwind defaults.

Reply:
Yes,i want this further in the future